src/ScreenshotWindow.vala

```diff
@@ -560,5 +569,11 @@ namespace Screenshot {
         private void cancel_clicked () {
             destroy ();
         }
+
+        // Save main window position so that this position can be used
+        // when the window reappears again
+        private void remember_window_position () {
```

Comment:
Please check if the `get_window ()` doesn't return `null` so you don't fail calling `get_root_origin`.